packages/protocol/contracts/governance/Validators.sol

```diff
@@ -597,7 +594,6 @@ contract Validators is
    *         the last time the group was slashed.
    */
   function resetSlashingMultiplier() external nonReentrant {
```

Comment:
Made available on L2 to ensure that the value could be set to 1. 

Due to the contract size, it is difficult to add additional logic that would always return 1 on L2, without significantly rewriting the contract.

Reply:
Should we create task for this ?

Reply:
This is a function that is usually called by the validators. 

So if any validator is slashed at the L2 transition time, they will need to reset their multiplier like they would normally have to do on L1.